Server/ConcordiaCurriculumManager/Services/CourseGroupingService.cs

```diff
@@ -82,7 +82,7 @@ public async Task<ICollection<CourseGrouping>> GetCourseGroupingsLikeName(string
             throw new InvalidInputException("The course grouping name cannot be empty.");
         }
 
-        return await _courseGroupingRepository.GetCourseGroupingsLikeName(name);
+        return await _courseGroupingRepository.GetPublishedCourseGroupingsLikeName(name);
```

Comment:
Why was this switched to published groupings only? Users on the frontend should still be able to search through unpublished groupings (published is distinct from approved).

Reply:
This was added since given that the dossier modification request uses this method to show the user what they can change. This is similar to how when we request to modify a course, we compare it with the latest published version and not the latest version.

Reply:
This is going to be switched to the service-layer `GetCourseGroupingsByDossierAndName` method Tony wrote

Reply:
When we create a course modification request, I don't see any references to the published version. Which line of code are you referring to specifically? I think in general we try to keep questions of published/unpublished out of the versioning system we have.